pipeline/outputs/stackdriver_special_fields.md

```diff
@@ -301,75 +196,41 @@ the logEntry will be:
 ```text
 {
     "jsonPayload": {
+        "logging.googleapis.com/operation": {
+            "extra1": "some string",
+            "extra2": 123,
+            "extra3": true
+        }
         ...
     }
-    "httpRequest": {
-        "requestMethod":"GET",
-        "requestUrl":"logging.googleapis.com",
-        "requestSize":"12",
-        "status":200,
-        "responseSize":"12",
-        "userAgent":"Mozilla",
-        "remoteIp":"255.0.0.1",
-        "serverIp":"255.0.0.1",
-        "referer":"referer",
-        "latency":"1s",
-        "cacheLookup":true,
-        "cacheHit":true,
-        "cacheValidatedWithOriginServer":true,
-        "cacheFillBytes":"12",
-        "protocol":"HTTP/1.2"
+    "operation": {
+        "id": "test_id",
+        "producer": "test_producer",
+        "first": true,
+        "last": true
     }
     ...
 }
 ```
 
-## Trace
+### Exceptions
 
-TraceId is resource name of the trace associated with the log entry.
-If enable autoformat_stackdriver_trace flag in config the entry will automatically get the projectID from the Google Metadata server and add it.
+#### Monitored Resource ID
 
-The JSON representation is as followed:
-```text
-"trace": string
-```
+The `logging.googleapis.com/monitored_resource` field is parsed in a special way, meaning it has some important exceptions:
 
-## Span ID
+The `type` field from the [`MonitoredResource` proto]() is not parsed out of the special field. It is read from the [`resource` plugin configuration option](https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver#configuration-parameters). If it is supplied in the `logging.googleapis.com/monitored_resource` special field, it will not be recognized.
 
-The span ID within the trace associated with the log entry.
+The `labels` field is expected to be an `object<string, string>`. If any fields have a value that is not a string, the value will be ignored and **not preserved**. The plugin will log an error and drop the field.
```

Comment:
```suggestion
The `labels` field is expected to be an `object<string, string>`. If any fields have a value that is not a string, the value will be ignored and not preserved. The plugin logs an error and drops the field.
```

Reply:
Fixed in [f2f67cf](https://github.com/fluent/fluent-bit-docs/pull/1585/commits/f2f67cf3cd9d5af702c78466174accaa9e401754)